ios/Classes/FacebookLikeButton.m

```diff
@@ -50,63 +55,56 @@ -(void)setForegroundColor_:(id)colorVal
 -(void)setLikeViewStyle_:(id)styleStr
 {
     ENSURE_SINGLE_ARG(styleStr, NSString);
-    FBLikeControl *btn = [self like];
+    FBSDKLikeControl *btn = [self like];
     NSString* style = [TiUtils stringValue:styleStr];
     if ([style isEqualToString:@"box_count"]) {
-        btn.likeControlStyle = FBLikeControlStyleBoxCount;
+        btn.likeControlStyle = FBSDKLikeControlStyleBoxCount;
     } else if ([style isEqualToString:@"button"]) {
-        btn.likeControlStyle = FBLikeControlStyleButton;
+        DEPRECATED_REMOVED(@"facebook.likeButton.likeViewStyle.button", @"5.0.0", @"5.0.0");
+        btn.likeControlStyle = FBSDKLikeControlStyleStandard;
     } else {
-        btn.likeControlStyle = FBLikeControlStyleStandard;
+        btn.likeControlStyle = FBSDKLikeControlStyleStandard;
     }
 }
 
 -(void)setAuxiliaryViewPosition_:(id)positionStr
 {
     ENSURE_SINGLE_ARG(positionStr, NSString);
-    FBLikeControl *btn = [self like];
+    FBSDKLikeControl *btn = [self like];
     NSString* position = [TiUtils stringValue:positionStr];
     if ([position isEqualToString:@"bottom"]) {
```

Comment:
it's here: https://github.com/appcelerator-modules/ti.facebook/blob/master/apidoc/LikeButton.yml#L77